src/pps/pps.proto

```diff
@@ -464,7 +464,9 @@ message PipelineInfo {
     Determined determined = 36;
   }
   Details details = 12;
-  string details_json = 13;
+  reserved "details_json";
+  string user_spec_json = 13;
```

Comment:
Why reuse the field number? Is it because by chance we are not using it anywhere in our code, so it's safe to re-use?

Reply:
Really, it’s renaming the field. `details_json` was less correct than `user_spec_json`. I’d be fine renumbering it too, to be honest.

Reply:
No worries, just asking for my own understanding.